tests/test/paywall.test.js

```diff
@@ -1,4 +1,4 @@
-const url = require('../helpers/url')
+const { main: url } = require('../helpers/url')
```

Comment:
any reason you use a different syntax for this one vs. home.test.js or dashboard.test.js?

Reply:
no, I hadn't noticed that.

Reply:
self reviews ;)